ideascube/mediacenter/views.py

```diff
@@ -50,6 +59,19 @@ def get_context_data(self, **kwargs):
         self._set_available_kinds(context)
         self._set_available_langs(context)
         self._set_available_tags(context)
+        # We assume that if there is a source, it is a package_id.
```

Comment:
Currently that's the only thing we use the `source` for, but is that going to be a safe assumption going forward?

Are we already planning to use the `source` for something else?